frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -242,36 +262,38 @@ export const ClusterVersionDetailsTable: React.SFC<ClusterVersionDetailsTablePro
     <>
       <div className="co-m-pane__body">
         <div className="co-m-pane__body-group">
-          <div className="co-detail-table co-detail-table--lg">
-            <div className="co-detail-table__row row">
-              <div className="co-detail-table__section col-sm-4 col-md-3">
-                <dl className="co-m-pane__details">
-                  <dt className="co-detail-table__section-header">Channel</dt>
-                  <dd>
-                    <CurrentChannel cv={cv} />
-                  </dd>
-                </dl>
-              </div>
-              <div className="co-detail-table__section col-sm-4 col-md-4">
-                <dl className="co-m-pane__details">
-                  <dt className="co-detail-table__section-header">
+          <div className="co-cluster-settings">
+            <div className="co-cluster-settings__row">
+              <div className="co-cluster-settings__section co-cluster-settings__section--current">
+                <dl className="co-m-pane__details co-cluster-settings__details">
+                  <dt>
                     <CurrentVersionHeader cv={cv} />
                   </dt>
                   <dd>
-                    <div>
-                      <CurrentVersion cv={cv} />
-                    </div>
-                    <UpdateLink cv={cv} />
+                    <CurrentVersion cv={cv} />
                   </dd>
                 </dl>
               </div>
-              <div className="co-detail-table__section col-sm-4 col-md-4">
-                <dl className="co-m-pane__details">
-                  <dt className="co-detail-table__section-header">Update Status</dt>
-                  <dd>
-                    <UpdateStatus cv={cv} />
-                  </dd>
-                </dl>
+              <div className="co-cluster-settings__section">
+                <div className="co-cluster-settings__row co-cluster-settings__row--status">
+                  <dl className="co-m-pane__details co-cluster-settings__details co-cluster-settings__details--status">
+                    <dt>Update Status</dt>
+                    <dd>
+                      <UpdateStatus cv={cv} />
+                    </dd>
+                  </dl>
+                  <div className="co-cluster-settings__row">
+                    <dl className="co-m-pane__details co-cluster-settings__details">
+                      <dt>Channel</dt>
+                      <dd>
```

Comment:
Is there a vert. gridline between 'Update Status' and 'Channel'?  Mockups I have do not show one:
![image](https://user-images.githubusercontent.com/12733153/84390311-60d99280-abc5-11ea-924f-8f2b7148c1f6.png)

Reply:
There is not.  This layout is to keep `Channel` and `Update` grouped together.  Without it, there would be an equal separation between `Update Status` and `Channel` and `Update`, which is not what the designs seem to indicate.